README.md

```diff
@@ -53,15 +53,22 @@ To get started quickly:
     cd learn-druid
    ```
 
-3. Launch the "Learn Druid" Docker environment:
+   To refresh your local copy with the latest notebooks:
+
+   ```bash
+   git restore .
+   git pull
+   ```
+
+4. Launch the "Learn Druid" Docker environment:
 
    ```bash
    docker compose --profile druid-jupyter up -d
```

Comment:
I just noticed we're starting here with "druid-jupyter" profile... I think we should instruct them to use "all-services" because that is what will get all the notebooks working.